src/core/core.scale.js

```diff
@@ -237,17 +237,17 @@ class Scale extends Element {
 
 		// implements box
 		/** @type {number} */
-		this.top = undefined;
+		this.top = -1;
```

Comment:
You can remove the type comments, it knows `-1` is number

Reply:
done